keep/api/models/alert.py

```diff
@@ -19,6 +19,9 @@
 from sqlalchemy import desc
 from sqlmodel import col
 
+if TYPE_CHECKING:
+    from keep.api.models.db.alert import Incident
```

Comment:
what is it?

Reply:
It's import for type checking `from_db_incident` method of `IncidentDto`. have to be this way to avoid import cycle.
https://github.com/keephq/keep/blob/d9db4882d6170a8c07c07feab2a12221aa63751b/keep/api/models/alert.py#L467

Reply:
TIL